app/src/debug/AndroidManifest.xml

```diff
@@ -5,8 +5,10 @@
           android:installLocation="auto">
 
   <application
+      android:icon="@drawable/debug_icon"
+      android:label="Quran Debug"
```

Comment:
This will only be visible in application manager in Settings app. Does not effect the title in launcher.